CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to [path-intersection](https://github.com/bpmn-io/path-inter
 
 ___Note:__ Yet to be released changes appear here._
 
+## 3.0.0
+
+* `FEAT`: turn into module
```

Comment:
Breaking changes missing. We could copy from https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#100

Reply:
```suggestion
* `FEAT`: turn into module

### Breaking Changes

* You must now configure a module transpiler such as Babel or Webpack in order to import from non-ES modules code.
```